compiler/luci/pass/src/FuseMulWithDivPass.cpp

```diff
@@ -98,10 +98,14 @@ luci::CircleConst *create_mul_const_with_new_value(luci::CircleConst *mul_const,
  *          where Scalar_new_Div_Const = Scalar_Div_Const / Scalar_Mul_Const
  *
  **/
-bool fuse_mul_with_div(luci::CircleDiv *div)
+bool fuse_mul_with_div_to_div(luci::CircleDiv *div)
 {
-  auto div_const = dynamic_cast<luci::CircleConst *>(div->x());
-  if (not div_const)
+  if (div->fusedActivationFunction() != luci::FusedActFunc::NONE)
```

Comment:
(optional) I think this is an optional condition. Mul's activation has to be NONE, but Div's can be anything.

Reply:
Thanks for pointing this :)
I wasn't sure about this but anyway we can revise for other activation later.